rma/models/rma_order_line.py

```diff
@@ -478,7 +478,7 @@ def action_rma_to_approve(self):
     @api.multi
     def action_rma_draft(self):
         if self.in_shipment_count or self.out_shipment_count:
-            raise UserError(_(
+            raise ValidationError(_(
```

Comment:
why not?

Reply:
because these errors happen when filling the form. that's why I think they should be UserError, because the user put wrong data or miss to fill something. ValidationError is for constrains, where the system is validating the data when saving/writing.

Reply:
Well, here the user is trying to change the state to draft, the data is correct.

Reply:
well, but the above one (the minimized one) is an onchange

Reply:
Ok, thanks for the explanation. Just another question. For the user perspective is there any difference?